changelog.md

```diff
@@ -1,3 +1,12 @@
+# 3.11.0
```

Comment:
Does changing the dependencies warrant a minor bump?
Are we as a package providing any new features by upgrading?

/cc @janmeier